perllib/FixMyStreet/App/Controller/Offline.pm

```diff
@@ -22,6 +22,11 @@ Offline pages Catalyst Controller - service worker handling
 
 sub service_worker : Path("/service-worker.js") {
     my ($self, $c) = @_;
+    if (FixMyStreet->test_mode == 2) {
```

Comment:
I think this code might be clearer/more future-proof if there was a `FixMyStreet->browser_testing` (or `->cypress_testing`/something else) method that did these checks.

Reply:
I've changed it to `FixMyStreet->test_mode eq 'cypress'`